tests/pc/test_lag_2.py

```diff
@@ -12,6 +12,8 @@
 from tests.common.helpers.dut_ports import decode_dut_port_name
 from tests.common.helpers.dut_ports import get_duthost_with_name
 from tests.common.config_reload import config_reload
+from tests.common.helpers.dut_utils import ignore_t2_syslog_msgs
```

Comment:
Is this import "ignore_t2_syslog_msgs" valid and defined ?

Reply:
I removed the import for now.  It can be added when PR #6056 is merged.